src/App.tsx

```diff
@@ -1,61 +1,154 @@
+import { useState } from 'react';
 import './App.scss';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list